src/test/java/core/basesyntax/strategy/operationhandler/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.strategy.operationhandler.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static final String APPLE = "apple";
+    private static final String BANANA = "banana";
+    private static final int APPLE_QUANTITY = 40;
+    private static final int BANANA_QUANTITY = 50;
+    private static FruitTransaction transaction;
+    private static BalanceOperationHandler balanceOperationHandler;
```

Comment:
```suggestion
    private static OperationHandler balanceOperationHandler;
```
the same for other implementations test classes